plugins/auto-sizes/readme.txt

```diff
@@ -7,12 +7,11 @@ License:      GPLv2 or later
 License URI:  https://www.gnu.org/licenses/gpl-2.0.html
 Tags:         performance, images, auto-sizes
 
-Instructs browsers to automatically choose the right size for images.
+Improvements for WordPress' responsive image functionality.
```

Comment:
```suggestion
Improvements for responsive images in WordPress.
```

Avoids the debate between `WordPress'` and `WordPress's`. Then again, WordPress is implied.